pyhuman/requirements.txt

```diff
@@ -4,6 +4,7 @@ colorama
 configparser
 crayons
 idna
+pyautogui
```

Comment:
I think `pyautogui` should also be added to the `requirements.txt` at the root of repo.